src/ui/controls/shortcut.cpp

```diff
@@ -209,6 +209,27 @@ void CShortcut::Draw()
 
         DrawIcon(m_pos, m_dim, uv1, uv2);
     }
+    if ( (m_state & STATE_DAMAGE) && Math::Mod(m_time, 0.7f) >= 0.3f )
+    {
+      Math::Point uv1, uv2;
+      float   dp;
+
+      m_engine->SetTexture("textures/interface/button2.png");
```

Comment:
I don't like the fact that this uses a texture for just a solid color... but it should work for now